include/spirv-tools/instrument.hpp

```diff
@@ -170,7 +170,7 @@ static const int kInstMaxOutCnt = kInstStageOutCnt + 4;
 static const int kInstErrorBindlessBounds = 0;
 static const int kInstErrorBindlessUninit = 1;
 static const int kInstErrorBuffAddrUnallocRef = 2;
-static const int kInstErrorBindlessBuffOOB = 3;
+// Deleted: static const int kInstErrorBindlessBuffOOB = 3;
```

Comment:
Suggest to just delete the line.  It can be recovered from revision control if needed later.

Reply:
Thank you for the careful review! :)

Since this is a public header and I have deleted a previously used code, I did this so if there are problems between the instrumentation and the tool using this instrumentation, it is a little more clear what the problem is ie. that the constant was purposely removed.

Rather than delete this line, I am going to add another comment line explaining things better. Let me know if you still have a problem with this.

Reply:
Oh, so the value `3` is now reserved in the sense that you need backwards compatibility?

Reply:
In that case, is there a way for us to at some point remove this line? Or do we have to have this value reserved for ever?

Reply:
Incidentally, I still don't see the changes you are referring to :)

Reply:
Can you see the new line at 174?

The comment will eventually go away. It is just there for the transitional period.

Reply:
I was expecting this rationale in the comment.  Otherwise, looking at this code without the context of our conversation will tempt anyone to just get rid of the line.

Reply:
OK. I have added the rationale. Let me know if this is good.

Reply:
This change is breaking the validation layers which use the old value. Can we please leave the old value in place, add the new values, update downstream and _then_ remove the old value?